here/resources/fleet_telematics_find_sequence_response.json

```diff
@@ -0,0 +1,135 @@
+                },
+                {
+                    "id": "destination1",
+                    "lat": -25.644764,
+                    "lng": -49.158558,
+                    "sequence": 5,
+                    "estimatedArrival": null,
+                    "estimatedDeparture": "2020-05-23T06:15:58Z",
+                    "fulfilledConstraints": []
+                },
+                {
+                    "id": "end",
+                    "lat": -25.643787,
+                    "lng": -49.158607,
+                    "sequence": 6,
+                    "estimatedArrival": "2020-05-23T06:16:28Z",
+                    "estimatedDeparture": null,
+                    "fulfilledConstraints": []
+                }
+            ],
+            "distance": "1997075",
+            "time": "123492",
+            "interconnections": [
+                {
+                    "fromWaypoint": "start",
+                    "toWaypoint": "destination5",
+                    "distance": 439328.0,
+                    "time": 27905.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                },
+                {
+                    "fromWaypoint": "destination5",
+                    "toWaypoint": "destination4",
+                    "distance": 0.0,
+                    "time": 0.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                },
+                {
+                    "fromWaypoint": "destination4",
+                    "toWaypoint": "destination3",
+                    "distance": 722082.0,
+                    "time": 38154.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                },
+                {
+                    "fromWaypoint": "destination3",
+                    "toWaypoint": "destination2",
+                    "distance": 388025.0,
+                    "time": 22921.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                },
+                {
+                    "fromWaypoint": "destination2",
+                    "toWaypoint": "destination1",
+                    "distance": 447533.0,
+                    "time": 34482.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                },
+                {
+                    "fromWaypoint": "destination1",
+                    "toWaypoint": "end",
+                    "distance": 107.0,
+                    "time": 30.0,
+                    "rest": 0.0,
+                    "waiting": 0.0
+                }
+            ],
+            "description": "Targeted best time; without traffic",
+            "timeBreakdown": {
+                "driving": 123492,
+                "service": 0,
+                "rest": 0,
+                "waiting": 0
+            }
+        }
+    ],
+    "errors": [],
+    "processingTimeDesc": "921ms",
+    "responseCode": "200",
+    "warnings": null,
+    "requestId": null
+}
```

Comment:
Please add a new empty line in the end.

Reply:
Can you explain to me why there must be a blank line? Already added.

Reply:
Thanks for the changes 👍 There are different opinions about that I can suggest you to read the comments on https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file.